docs/articles/documentation/reference/configuration-file.md

```diff
@@ -1082,6 +1082,22 @@ The `disableMultipleWindows` option disables support for multi-window testing in
 *CLI*: [--disable-multiple-windows](command-line-interface.md#--disable-multiple-windows)  
 *API*: [runner.run({ disableMultipleWindows })](testcafe-api/runner/run.md)
 
+## retryTestPages
+
+If this option is enabled, TestCafe retries failed network requests for webpages visited during tests. The retry limit is ten times.
```

Comment:
see note above for the same phrasing

Reply:
see [comment](https://github.com/DevExpress/testcafe/pull/5831#discussion_r555659869)

Reply:
i replied above